templates/tom_targets/target_detail.html

```diff
@@ -78,10 +84,14 @@ <h4>Observations</h4>
       <div class="tab-pane in active" id="photometry">
         {% load dataproduct_extras %}
         {% mop_photometry target %}
-        </div>
+      </div>
       <div class="tab-pane" id="spectroscopy">
         {% spectroscopy_for_target target %}
       </div>
+      <div class="tab-pane" id="pylima-model">
+        {% load dataproduct_extras %}
```

Comment:
Since the dataproduct_extras are now required for both lines 79 and 92, this load can be moved to the general load at the top of this file.

Reply:
Done.